### ISSUE TITLE
Option to rebuiild a package

### ISSUE THREAD

**Issue by root2185** [trizen/trizen#257]
I have set trizen to move built packages in a folder where I have a local repo.

After deleting the package source dir, the local repo files and updating pacman databse trizen still tries to install the what was in the database.

**Comment by striczkof** [comment]
Yes please! This has been requested before on #86 but I agree that that itself is overcomplicated. A simple `--rebuild` option is good enough.

yay and paru both takes --rebuild options, would be awesome for trizen to do the same.
`paru -S --rebuild pkg`
`yay -S --rebuild pkg`
`trizen -S --rebuild pkg` (absolutely great if this gets implemented)

Since trizen is relatively simple enough. I can try making the PR myself in a bit.

**Comment by root2185** [comment]
#86 is different https://github.com/trizen/trizen/issues/86#issuecomment-358973378:

> rebuild is 1 thing, but I'm talking about rebuilding AUR packages dependent on updated packages
